metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -0,0 +1,26 @@
+=== windows perfmon MetricSet
+
+This is the perfmon metricset of the module windows.
+
+[float]
+=== Features and configuration
+
+Metricset to collect performance counters.
+Example configuration:
+```
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 10s
+  perfmon.counters:
+    - group: "processor"  
+      collectors:
+        - alias: "processor_perfomance"
```

Comment:
That's right. Group name is only for the event. To added to the alias field would be an option `disk.bytes_written`. But this assumes that the user have to added for all the counters he wants to group. `disk.counter1`, `disk.counter2`. My concerns are if you have a lot of counters in one group it's easier to make a mistake `disk.counter1`, `dsk.counter2`